sdk/helper/pluginutil/env.go

```diff
@@ -31,6 +31,10 @@ const (
 	// PluginCACertPEMEnv is an ENV name used for holding a CA PEM-encoded
 	// string. Used for testing.
 	PluginCACertPEMEnv = "VAULT_TESTING_PLUGIN_CA_PEM"
+
+	// PluginMultiplexingOptOut is a comma separated list of plugin names opted-out
```

Comment:
Is this comment still valid? Looks like we aren't using a comma separated list of plugin names any longer

Reply:
Ah okay, after going through the PR more I now understand. I think I got confused because of the flag's value, but looks like we can even have this be an empty string and the opt out would essentially still work the same, so I wonder if it makes sense to just have this be an empty string for now.

The doc string for this flag would explain why it is empty, since it is 'a comma separated list of plugin names opted-out'. If the list is empty then it becomes apparent that none of the plugins are opting out. Not a blocker for the PR, but curious to hear thoughts. The current flag value definitely threw me off a bit 😅

Reply:
I'm not sure I understand the comment :sweat_smile: 

This is a constant defining the env variable used to optionally set a list of plugins to opt out. I'll try to rephrase the comment let me know if it make more sense with:
```
// PluginMultiplexingOptOut is an ENV name used to define a comma separated list of plugin names
// opted-out of the multiplexing feature; for emergencies if multiplexing ever causes issues
```

Reply:
Misunderstanding on my part